qlib/log.py

```diff
@@ -166,7 +166,79 @@ def filter(self, record):
 
 
 def set_global_logger_level(level: int):
+    """set qlib.xxx logger handlers level
+
+    Parameters
+    ----------
+    level: int
+        logger level
+
+    Examples
+    ---------
+
+        .. code-block:: python
+
+            import qlib
+            import logging
+            from qlib.log import get_module_logger, set_global_logger_level
+            qlib.init()
+
+            tmp_logger_01 = get_module_logger("tmp_logger_01", level=logging.INFO)
+            tmp_logger_01.info("1. tmp_logger_01 info show")
+
+            global_level = logging.WARNING + 1
+            set_global_logger_level(global_level)
+            tmp_logger_02 = get_module_logger("tmp_logger_02", level=logging.INFO)
+            tmp_logger_02.log(msg="2. tmp_logger_02 log show", level=global_level)
+
+            tmp_logger_01.info("3. tmp_logger_01 info do not show")
+
+    """
+    qlib_logger = logging.root.manager.loggerDict.get("qlib", None)
+    if qlib_logger is not None:
+        for _handler in qlib_logger.handlers:
+            _handler.level = level
+
+
+@contextmanager
+def set_global_logger_level_contextmanager(level: int):
```

Comment:
The function name is too long...
You can use a shorter name  (e.g. use `cm` to replace contextmanger)